apps/elixir_ls_utils/priv/language_server.sh

```diff
@@ -13,8 +12,18 @@ readlink_f () {
   fi
 }
 
-SCRIPT=$(readlink_f $0)
-SCRIPTPATH=`dirname $SCRIPT`
-export ERL_LIBS="$SCRIPTPATH:$ERL_LIBS"
+version=$(elixir -v|grep ^Elixir| sed 's/Elixir \(.*\) (compiled with Erlang\/OTP \(.*\))/\1-\2/')
+
+target_dir=~/.cache/elixir-ls/$version
```

Comment:
Might want to respect XDG dirs here?

```suggestion
target_dir="${XDG_CACHE_HOME:-~/.cache}/elixir-ls/$version"
```